unittest/CMakeLists.txt

```diff
@@ -2,7 +2,7 @@ cmake_minimum_required(VERSION 3.15)
 
 project(md_unittest VERSION 0.1 LANGUAGES C)
 
-add_executable(md_unittest unittest.c)
+add_executable(md_unittest unittest.c "test_lammps.c")
```

Comment:
Just add it to the target_sources defined on line 12 instead. Perhaps it is correct you explicitly add dependencies for the executable as you did it here. But if that is the case, I'll just add all of the sources in there.

Reply:
The changes was done automatically by Visual Studio, didn't think much of it. I have made adjustments now so that it is in line with the rest of mdlib.